### PR TITLE
feat(database): import & export for custom schemas & endpoints

### PR DESCRIPTION
**Pull request by ChrisPdgn**
**What kind of change does this PR introduce?** 

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other (please describe)

**Does this PR introduce a breaking change?** 

- [ ] Yes
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `main` branch
- [ ] When resolving a specific issue, it's referenced in the PR's description  (e.g. `fix #xxx`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature

### CODE REVIEW

File: modules/database/src/admin/schema.admin.ts [hunk 1]
```diff
@@ -27,6 +27,50 @@ export class SchemaAdmin {
     private readonly customEndpointController: CustomEndpointController,
   ) {}
 
+  async exportCustomSchemas(): Promise<UnparsedRouterResponse> {
+    return await this.database
+      .getSchemaModel('_DeclaredSchema')
+      .model.findMany(
```
Comment (kon14) [inline modules/database/src/admin/schema.admin.ts:33]: Shouldn't this also retrieve internal schemas (SQL)

File: modules/database/src/admin/customEndpoints/customEndpoints.admin.ts [hunk 2]
```diff
@@ -31,6 +31,88 @@ export class CustomEndpointsAdmin {
...
+          delete obj.createdAt;
+          delete obj.updatedAt;
+          delete obj.__v;
+          return obj;
+        }),
+      );
+  }
+
+  async importCustomEndpoints(
+    call: ParsedRouterRequest,
```
Comment (kon14) [inline modules/database/src/admin/customEndpoints/customEndpoints.admin.ts:50]: `schemaId` entries should be updated as they won't match the exported ones

**Review by kon14 — CHANGES_REQUESTED** [review]

File: modules/database/src/admin/customEndpoints/customEndpoints.admin.ts (hunk 2 above)
Comment (ChrisPdgn) [reply modules/database/src/admin/customEndpoints/customEndpoints.admin.ts:50]: It is updated in line 75

File: modules/database/src/admin/index.ts [hunk 3]
```diff
@@ -53,6 +53,29 @@ export class AdminHandlers {
...
     this.routingManager.clear();
     // Schemas
+    this.routingManager.route(
+      {
+        path: '/schemas/export',
+        action: ConduitRouteActions.GET,
+        description: `Export custom schemas.`,
+      },
+      new ConduitRouteReturnDefinition('ExportSchemas', {
+        schemas: [ConduitString.Required],
```
Comment (kon14) [inline modules/database/src/admin/index.ts:63]: That's JSON.

**Review by kon14 — CHANGES_REQUESTED** [review]

**Review by kkopanidis — APPROVED** [review]

**Pull request merged**